app/population/view/dashboard.njk

```diff
@@ -0,0 +1,32 @@
+{% extends "layouts/collection.njk" %}
+
+{#{% block pageTitle %}#}
```

Comment:
Any reason this and the next block are commented out?

Reply:
It's a remanent of trying to fit it into the main dashboard screen - removed it.